src/main/java/org/gradle/profiler/mutations/support/ScenarioSupport.java

```diff
@@ -0,0 +1,33 @@
+package org.gradle.profiler.mutations.support;
+
+import com.typesafe.config.Config;
+import org.gradle.profiler.ConfigUtil;
+
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class ScenarioSupport {
+
+    public static List<File> sourceFiles(Config config, String scenarioName, File projectDir, String key) {
+        return ConfigUtil.strings(config, key)
+            .stream()
+            .map(fileName -> openFile(fileName, projectDir, scenarioName))
+            .filter(Objects::nonNull)
```

Comment:
You could move filtering for `null`s earlier, and then you can simplify `openFile()`. But why are we filtering for `null`? Does `ConfiguUtil.strings()` ever return a `null` entry?

Reply:
I moved nonull higher and removed open method